docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -258,7 +258,10 @@ username which you can search for to get the events history:
 
 ## Step 8. Using AWS CLI
 
-First, log into the previously configured AWS console app:
+Prequisites:
+- `aws` Command Line Interface (CLI) tool in PATH. How to [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
```

Comment:
I think this should be in the initial Prerequisites section so a user can do all of their outside-the-guide setup first (as much as possible), then stay in the guide until the end.

Reply:
I copied this to the top as well. I think good to have both in both places

Reply:
@ptgott please confirm this is good

Reply:
@stevenGravy Just added some quick suggestions

Reply:
Done, thanks @ptgott !